app/models/mixins.py

```diff
@@ -0,0 +1,31 @@
+import base64
+from sqlalchemy.dialects.postgresql import BIGINT
+
+from app import db
+from app.utils import random_positive_public_id
+
+
+class PublicIdMixin:
+    public_id = db.Column(BIGINT, default=random_positive_public_id, nullable=False, index=True, unique=True)
+
+    @staticmethod
+    def __pad_as_bytes(val):
+        """Converts an int to its equivalent in bytes and prepends with \x00 bytes until it's a multiple of 8 bytes
+        long to create ciphertexts of the same length."""
+        return val.to_bytes(8, 'big')
+
+    @staticmethod
+    def __append_equals(text):
+        """Appends '=' to the given text until it's a multiple of 8 characters long and so suitable for base32
+        decoding."""
+        return text + ''.join('=' * (4 - (len(text) % 4)))
+
+    @staticmethod
```

Comment:
prefer `@classmethod` cos then you can use `cls. __pad_as_bytes` rather than having to name the class again